snippets/csharp/VS_Snippets_CLR_System/system.DateTime.Ticks/cs/Ticks.cs

```diff
@@ -20,14 +20,16 @@ public static void Main()
       Console.WriteLine("   {0:N0} days, {1} hours, {2} minutes, {3} seconds", 
                         elapsedSpan.Days, elapsedSpan.Hours, 
                         elapsedSpan.Minutes, elapsedSpan.Seconds);
-      // If run on December 14, 2007, at 15:23, this example displays the
+
+      // If run on November 14, 2019, at 6:21pm, this example displays the
```

Comment:
I think we can simplify that without giving specific times

```suggestion
      // This example displays an output similar to the following:
```